drivers/can/can_mcux_flexcan.c

```diff
@@ -530,11 +530,29 @@ DEVICE_AND_API_INIT(can_mcux_flexcan_0, DT_INST_0_NXP_KINETIS_FLEXCAN_LABEL,
 
 static void mcux_flexcan_config_func_0(struct device *dev)
 {
+#if defined(CONFIG_SOC_SERIES_KINETIS_K6X)
```

Comment:
I suggest `#ifdef`ing on whether or not the specific IRQ is defined here - that makes for a more flexible driver.

Reply:
Done.